ecs/firehose.conf

```diff
@@ -0,0 +1,17 @@
+[INPUT]
```

Comment:
One thing I forgot to mention, we should add a service section with grace period set to 30:

```
[SERVICE]
     Grace 30
```